triggertask/spring-cloud-starter-stream-source-triggertask/src/main/java/org/springframework/cloud/stream/app/triggertask/source/TaskPayloadProperties.java

```diff
@@ -63,12 +68,22 @@ public void setUri(String uri) {
 		this.uri = uri;
 	}
 
-	public String getProperties() {
-		return properties;
+	public String getEnvironmentProperties() {
+		return environmentProperties;
+	}
+
+	public void setEnvironmentProperties(String environmentProperties) {
+		Assert.notNull(environmentProperties, "environmentProperties must not be null");
```

Comment:
Given both the `environmentProperties` and `deploymentProperties` are initialized with empty string as the default, does this null check necessary?